Example/Example/MoreAttachmentRender.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  MoreAttachmentRender.swift
+//  AztecExample
+//
+//  Created by Jorge Leandro Perez on 3/22/17.
+//  Copyright © 2017 Automattic Inc. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import Aztec
+
+
+// MARK: - MoreAttachmentRender: Renders More Comments!
+//
+class MoreAttachmentRender {
```

Comment:
Can we rename these classes to `Renderer` (the one that renders) instead of `Render` (a render operation)?